Mathlib/AlgebraicGeometry/Gluing.lean

```diff
@@ -132,8 +132,8 @@ def gluedScheme : Scheme := by
   swap
   · exact (D.U i).affineCover.map y
   constructor
-  · dsimp [-Set.mem_range]
-    rw [coe_comp, Set.range_comp]
+  · dsimp
+    erw [coe_comp, Set.range_comp]
```

Comment:
This regression is caused by `Spec.topObj_forget`, which enables removing the magical `change` in `GammaSpecAdjunction`. I think it's a net improvement, and hopefully we can fix this `erw` independently later.

Reply:
Does `dsimp [-Spec.topObj_forget]` work?

Reply:
It does. I'm using that now.

It would be great if we could add further simp lemmas so that even with `topObj_forget` we still end up in a nice state where `coe_comp` can fire, but I don't see it.